api/server/sdk/node.go

```diff
@@ -182,3 +182,19 @@ func (s *NodeServer) EnumerateNodeDrainJobs(
 
 	return s.cluster().EnumerateNodeDrainJobs(ctx, req)
 }
+
+func (s *NodeServer) VolumeUsageByNode(
+	ctx context.Context,
+	req *api.SdkVolumeUsageByNodeRequest,
+) (*api.SdkVolumeUsageByNodeResponse, error) {
+
```

Comment:
There needs to be checks here for `.driver()` could return `nil` if the SDK objects are not ready. Check other code for examples.

Reply:
thanks!.  Fixed.